Common/src/config_structure.cpp

```diff
@@ -4633,11 +4687,14 @@ void CConfig::SetOutput(unsigned short val_software, unsigned short val_izone) {
       case INVERSE_VOLUME:
         cout << "Cell stiffness scaled by inverse of the cell volume." << endl;
         break;
-      case WALL_DISTANCE:
-        cout << "Cell stiffness scaled by distance from the deforming surface." << endl;
+      case DEF_WALL_DISTANCE:
+        cout << "Cell stiffness scaled by distance to nearest deforming surface." << endl;
+        break;
+      case SOLID_WALL_DISTANCE:
```

Comment:
My suggestion is to eliminate in the future the DEF_WALL_DISTANCE... there is not a good reason to keep that option. I kept that option because the regression test... but, for the next release my recommendation is to update all the regression tests and use only the distance to the SOLID_WALL. And yes... there is a WALL_distance in the limiter stuff, in the future it could be important to rename all the limiters adding the "_LIM" keyword to the name.